src/app/api/media-content/index.ts

```diff
@@ -21,10 +21,13 @@ export default async function fetchMediaContent_(request: any) {
     course_id: z.string().optional(),
     unit_id: z.string().optional(),
     topic_id: z.string().optional(),
+    grade_id: z.string().optional(),
```

Comment:
@mikemtdev  I noticed you might be working on the backend for too.

While I'm happy to offer assistance if needed, I wanted to highlight the importance of maintaining clear documentation for future updates.

Unsolicited changes to the backend code can sometimes make it difficult to keep the frontend and backend in sync.

To ensure smooth collaboration and long-term maintainability, would you be open to discussing any planned backend changes before implementation? This way, we can keep the documentation up-to-date and avoid potential issues down the line.

Let me know your thoughts, and I'm happy to chat further!

Reply:
@OlivierJM I am currently documenting end points in postman. @mikemtdev What do you need from the backend with regards to this `PR` ?